src/styles/components/buttons.css

```diff
@@ -9,55 +9,68 @@
     &[disabled] {
       @apply cursor-not-allowed;
     }
-   
-    &.text {
-      @apply text-blue;
-      @apply hover:text-blue-dark;
+
+    &.primary {
+      @apply bg-gradient-to-r from-purple to-purple-dark text-white;
 
       &[disabled] {
-        @apply text-gray-lighter;
+        @apply bg-gradient-to-r from-white-ish to-white-ish text-gray-lightest;
       }
+    }
+
+    &.outline-primary {
+      @apply border-2 border-primary text-primary;
 
+      &[disabled] {
+        @apply border-gray-lightest text-gray-lightest;
+      }
     }
 
-    &.primary {
-      @apply bg-blue text-white;
-      @apply hover:bg-blue-crayola;
+    &.yellow {
+      @apply bg-yellow text-primary;
```

Comment:
There are going to be a new primary button? I'm trying to understand if you will replace `primary` for `yellow.` Why don't you replace de `primary` class with the yellow `properties` if that is the case?

Reply:
I think this happened because I reordered the colors list. Yes, the primary color is going to change but to another one which is already defined as primary in tailwind.config.js :)